godot-macros/src/method_common.rs

```diff
@@ -0,0 +1,179 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+use proc_macro2::TokenStream as TokenStream2;
+use quote::quote;
+use syn::{
+    parse::{Parse, ParseStream},
+    token::Comma,
+    Ident, Result, ReturnType, Signature, Type,
+};
```

Comment:
Please avoid nested `use` statements, just use multiple lines (here it doesn't matter because `syn` will be removed, but in other places).